services/ui-src/src/components/reports/ModalDrawerReportPage.tsx

```diff
@@ -66,9 +66,8 @@ export const ModalDrawerReportPage = ({ route, validateOnRender }: Props) => {
   // create drawerForm from json
   const drawerForm = { ...drawerFormJson };
 
-  const drawerTitleText = `${verbiage.drawerTitle} ${
-    !selectedEntity?.isRequired && `Other: `
-  }${selectedEntityName}`;
+  const otherText = !selectedEntity?.isRequired ? `Other: ` : `Other`;
```

Comment:
i think it might be "Other: " : ""; because it's doing this right now in deploy?

![Screenshot 2023-11-29 at 2 13 22 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/127151429/e40aed14-d816-4ac2-993f-5a16310b4580)

Reply:
![giphy](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/fee80f75-6dfd-40f0-9851-a0f22a4e7964)